blog/_posts/2020-05-26-create-your-own-theme-detailed-walkthrough.md

```diff
@@ -0,0 +1,372 @@
+
+Then copy past an existing theme :
+
+`cp high-contrast_color-scheme-light.css mytheme.css`
+
+The name of your file is not important, the theme name will be included in the header of the CSS file.
+
+`vi mytheme.css`
+
+Choose your theme name :
+
+``` /*
+ * name: Hacker */ ```
+
+When you go to your convos instance settings, you will already be able to select your new theme (maybe you will need to
+refresh your page) :
+
+![](/public/screenshots/2020-05-26-choosenewtheme.png)
+
+We created our theme, and moved on it. At the moment, our new theme is a vanilla copy of `high-contrast light` but it's
+a start and we can now play with it (and it will be reflected in our convos in almost real time).
+
+**It is very easy to develop a new theme because without even having to restart convos, your changes are immediately
+viewable !!!**
+
+If it is your first theme, I recommend a light theme (bg light and text dark), it is easier.
+
+
+## How do I choose the colors ?
+
+Honestly I first started by typing **"hacker theme green"** in [google image](https://www.google.com/search?q=hacker+theme+green&tbm=isch&ved=2ahUKEwjljK2Zwc_pAhUMHBoKHU3MDPQQ2-cCegQIABAA&oq=hacker+theme+green&gs_lcp=CgNpbWcQAzoCCCk6BAgAEBM6AggAOgQIABAeOgYIABAIEB5QnwlYhxVgmRdoAXAAeACAAWaIAfQEkgEDNS4ymAEAoAEBqgELZ3dzLXdpei1pbWc&sclient=img&ei=V_zLXqWFE4y4aM2Ys6AP&bih=771&biw=1533&client=ubuntu&hs=Vge&safe=active) and I looked at results.
+
+![](/public/screenshots/2020-05-26-google.png)
+
+It gave me an idea about what colors to put together : 
+* First the green is really flashy, let’s go for flashy then.
+* There is black background or eventually dark grey background (cool, I will try to put this grey on one panel, I
+  usually don't like having all convos panel the same color)
+* Text colors are usually green or grey
+
+To have the color of the grey used in one of the screenshots (the grey in the treeview renders well with overall black
+bg) :
+
+![](/public/screenshots/2020-05-26-grey.png)
+
+I use the *"eye dropper"* from [gimp](https://www.gimp.org/downloads/) :
+
+![](/public/screenshots/2020-05-26-gimp.png)
+
+I used it to click magically on the color that I needed to know the value...
+
+...And I get the html notation `#181818` that you will see later in the CSS : 
+
+![](/public/screenshots/2020-05-26-htmlnotation.png)
+
+
+## Panel backgrounds 
+
+Ok let's start changing the colors. 
+
+It is a **hacker** theme, it has to be daaark ! 
+
+Therefore I wanted to put black on left/mid panel and grey on right panel... I was actually not sure about how to
+arrange this, I made some tests and I finally used black everywhere !
+
+Then look at our first modification.
+
+``` /* Backgrounds */ --sidebar-left-bg: black; --body-bg: black; --sidebar-right-bg: black;
+
+```
+
+I can use color shortcuts names (`black`) or hex values (`#RRGGBB` from `#000000` to `#ffffff`)
+
+You can use for instance an [online color picker](https://www.w3schools.com/colors/colors_picker.asp) to find colors
```

Comment:
I'm sorry, but I can't link to w3schools.

Reply:
Honestly there is no exact alternative for this link, I changed to something close